src/main/java/org/prebid/server/spring/config/WebConfiguration.java

```diff
@@ -164,6 +166,7 @@ Router router(CookieHandler cookieHandler,
         router.get("/bidders/params").handler(bidderParamHandler);
         router.get("/info/bidders").handler(biddersHandler);
         router.get("/info/bidders/:bidderName").handler(bidderDetailsHandler);
+        router.post("/event").handler(notificationEventHandler);
```

Comment:
Sorry, it is implicit but it should be `GET` request. Fixed in the tech spec to explicitly say this.